### ISSUE TITLE
Platform version '43.0.4' is not compatible

### ISSUE THREAD

**Issue by iqbalhusen** [prikhi/pencil#791]
I am getting following error on Ubuntu 14.04 LTS.

```
Error: Platform version '43.0.4' is not compatible with
minVersion >= 36.0
maxVersion <= 42.*
```

**Comment by manuelvillar** [comment]
I get the same error on Debian Sid with FF 43.0.4

**Comment by iqbalhusen** [comment]
I found out that `43.0.4` is the version number of Firefox installed on my computer. I think downgrading Firefox to a lower version (e.g. 42.0) will solve this. But I don't know why this repo has not been updated to work on latest Firefox version.

**Comment by mnagel** [comment]
this issue  #791 should be closed as duplicate of #777
